ci/do-audit.sh

```diff
@@ -30,6 +30,7 @@ cargo_audit_ignores=(
   --ignore RUSTSEC-2023-0001
 
   --ignore RUSTSEC-2022-0093
+  --ignore RUSTSEC-2024-0003
```

Comment:
nit: This has been addressed upstream so you can rebase and remove this

Reply:
done.